pkg/windows/windows.go

```diff
@@ -71,8 +71,8 @@ const (
 	windowsExporterServiceArgs = "--collectors.enabled " +
 		"cpu,cs,logical_disk,net,os,service,system,textfile,container,memory,cpu_info"
 	// remotePowerShellCmdPrefix holds the PowerShell prefix that needs to be prefixed  for every remote PowerShell
-	// command executed on the remote Windows VM
-	remotePowerShellCmdPrefix = "powershell.exe -NonInteractive -ExecutionPolicy Bypass "
+	// command executed on the remote Windows VM. Note that the opening double quote should be closed when using this.
+	remotePowerShellCmdPrefix = "powershell.exe -NonInteractive -ExecutionPolicy Bypass \""
```

Comment:
I'd prefer you improve this by adding a new function such as `formatRemotePowerShellCommand(string) string`
which wraps the given string in this prefix and adds the quote at the end.

the trailing quote requirement adds complexity and contributor's/reviewers lives slightly harder

Reply:
+1